megamek/src/megamek/logging/MMLogger.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * MegaMek
+ * Copyright (c) 2024 - The MegaMek Team. All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+package megamek.logging;
+
+import javax.swing.JOptionPane;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import io.sentry.Sentry;
+
+public class MMLogger {
+  MMLogger() {
+    throw new IllegalStateException("MMLogger Utility Class");
+  }
+
+  public static void info(String message) {
+    Logger logger = LogManager.getLogger();
+
+    if (logger.isInfoEnabled()) {
```

Comment:
this suggests that you plan to replace all current info() calls with this one, right? Because it doesnt make sense to check if info is enabled for some but not all info logs. In that case, would it make sense to really replace *all* logger calls? And if so, we'd need to have an error() call without an exception parameter because not all errors come from an exception.

Reply:
That is the intention and I can add the other error method.

But the number of places that would need to be adjusted for this, I'm not going to do it all at once and only a few files at a time. A slow migration to this system.

Reply:
I misphrased that. It's not that I expect you to actually do any replacing, I only meant how things would ideally be. Mass-replacing will only create merge conflicts. As there is no intolerable downside to keeping LogManager calls, we can leave it to whoever touches classes to switch to MMLogger.

Reply:
Yea, my intention is to do 5-10 files at a time for Sentry and just replace the logs as I go to keep it uniform. It'll take a few months just for MM a few more for MML and a few more for MHQ. I figure by the time we reach the start of J21 testing I'll finally be done adding this in everywhere.